src/hoplon/core.cljs

```diff
@@ -534,8 +535,9 @@
 ;; HTML Elements ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 (defn html [& args]
  "Updates and returns the document's `html` element in place."
- (let [elem (mksingleton "documentElement")]
-   (elem (first (parse-args args)))))
+  (let [elem (mksingleton "documentElement")
+        [attr kids] (parse-args args)]
+   (elem kids)))
```

Comment:
I'm maintaining Hoplon for a couple of months now and finally reproduced this problem. Thank you for submitting this PR.
I'm not sure if you remember why this change was necessary, looks like previously we passed only the `args` to the element, and now we pass only the `kids`.
Let me know if you are interested in getting this merged, I will totally understand if this is not the case anymore.

Reply:
I have a few guesses, but I can't quite remember the reasoning for this exact approach. Happy to see it fixed in your other PR!